hadoop-ozone/dev-support/checks/hadolint.sh

```diff
@@ -17,25 +17,24 @@ DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
 cd "$DIR/../../.." || exit 1
 REPO_DIR="$DIR/../../.."
 
-ERROR=0
-
-
-for Dockerfile in $(find "$REPO_DIR/hadoop-ozone" "$REPO_DIR/hadoop-hdds" -name Dockerfile | sort); do
-  echo "Checking $Dockerfile"
+REPORT_DIR=${OUTPUT_DIR:-"$DIR/../../../target/hadolint"}
+mkdir -p "$REPORT_DIR"
+REPORT_FILE="$REPORT_DIR/summary.txt"
 
+for Dockerfile in $(find hadoop-ozone hadoop-hdds -name Dockerfile | sort); do
   result=$( hadolint $Dockerfile )
   if [ ! -z "$result" ]
   then
-    echo "$result"
-    echo ""
-    ERROR=1
+    echo "$result" | tee -a "$REPORT_FILE"
   fi
```

Comment:
Loop body can be simplified to:

```
  hadolint $Dockerfile | tee -a "$REPORT_FILE"
```

since we no longer need to set `ERROR` at the same time.